imsv-docs-docusaurus/openapi/endpoints/kyc/get-kyc-profile.yaml

```diff
@@ -25,8 +31,11 @@ responses:
         **FORBIDDEN**
         Account does not exist or there are no sufficient permissions to view KYC profile for this account.
 
-        **KYC_PROFILE_DOES_NOT_EXIST** 
+        **KYC_PROFILE_DOES_NOT_EXIST**
         KYC profile does not exist for this account. It gets created upon initiating the first KYC check.
+
+        **LIVENESS_MISMATCH**
```

Comment:
Just curious -- do we actually return this error code? Normally when a resource is requested from the wrong domain then we return 403 Forbidden.

Reply:
we do actually return this error code, I checked using unit tests and by doing a static code analysis